src/helpers.js

```diff
@@ -26,49 +26,103 @@ const { objToString } = require('./utils');
 
 const { DEV_CONSOLE_BASE_URL, DEV_CONSOLE_API_KEY, AIO_CLI_API_KEY } = CONSTANTS;
 
+async function getDevConsoleConfigFromFile(configFilePath) {
+	try {
+		if (!fs.existsSync(configFilePath)) {
+			throw new Error(
+				`Config file does not exist. Please run the command: aio config:set api-mesh.configPath <path_to_json_file> with a valid file.`,
+			);
+		}
+
+		const data = JSON.parse(fs.readFileSync(configFilePath, { encoding: 'utf8', flag: 'r' }));
+
+		if (!data.baseUrl || !data.apiKey) {
+			throw new Error(
+				'Invalid config file. Please validate the file contents and try again. Config file must contain baseUrl and apiKey.',
+			);
+		}
+
+		const baseUrl = data.baseUrl.endsWith('/')
+			? data.baseUrl.slice(0, data.baseUrl.length - 1)
+			: data.baseUrl;
+
+		const config = {
+			baseUrl: baseUrl,
+			accessToken: (await getLibConsoleCLI()).accessToken,
+			apiKey: data.apiKey,
+		};
+
+		logger.debug(`Using cli config from ${configFilePath}: ${objToString(config)}`);
+
+		return config;
+	} catch (error) {
+		logger.error(
+			'Please run the command: aio config:set api-mesh.configPath <path_to_json_file> with a valid config file.',
+		);
+
+		throw new Error(error);
+	}
+}
+
+async function getDevConsoleConfigFromObject(configObject) {
+	const { baseUrl, apiKey } = configObject;
+	const config = {
+		baseUrl,
+		accessToken: (await getLibConsoleCLI()).accessToken,
+		apiKey,
+	};
+
+	logger.debug(`Using cli config: ${objToString(config)}`);
+
+	return config;
+}
+
 /**
  * @returns {any} Returns a config object or null
  */
 async function getDevConsoleConfig() {
-	const configFile = Config.get('api-mesh.configPath');
+	const configFileOrObject = Config.get('api-mesh.cliConfig');
+
+	/**
+	 * Old legacy option, needs to be deprecated
+	 */
+	const configPath = Config.get('api-mesh.configPath');
+	if (configPath) {
+		if (configFileOrObject) {
+			throw new Error(
+				'Found both cliConfig and configPath in api-mesh config. Please consider using only cliConfig.',
+			);
+		} else {
+			console.warn(
```

Comment:
@revanth0212, just curious, since we have eslint rules to prevent the usage of console, will this throw some error that console is not supported or something like that?

Reply:
nvm, just found out that we are disabling the no -console rule

Reply:
Good point deepty. Had to put it there along with logger because, people can switch logger off and when that happens, they ll have no clue what happened.

We can and should standardize this logger/console stuff for sure.